src/l500/l500-serializable.h

```diff
@@ -0,0 +1,23 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2018 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include "serializable.h"
+#include "hw-monitor.h"
+#include "sensor.h"
+
+namespace librealsense
+{
+    class l500_serializable_base : public serializable_interface
```

Comment:
Probably not required , can reuse the serializable interface/implementation for D400